### ISSUE TITLE
Ability to use karma.conf.js

### ISSUE THREAD

**Issue by Luxor** [craigburke/karma-gradle#18]
Currently it is generated by plugin. In case of migrating from old project with big karma conf file. It will be helpfull

**Comment by craigburke** [comment]
Makes sense. I can check the root of the project for a karma.conf.js file.
